BUILD.gn

```diff
@@ -132,6 +132,7 @@ if (current_toolchain != "${dir_pw_toolchain}/default:default") {
       deps = [
         "${chip_root}/scripts:matter_yamltests_distribution.wheel",
         "${chip_root}/scripts/py_matter_yamltests:matter-yamltests.wheel",
+        "${chip_root}/src/controller/python:matter-core-module.wheel",
```

Comment:
follow up for me - either add all the -core -cluster -repl ones here or remove this line